docs/root/start/install.rst

```diff
@@ -19,16 +19,16 @@ Install Envoy on Debian-based Linux
 
    .. code-tab:: console Debian bookworm
 
-      $ wget https://apt.envoyproxy.io/signing.key  | sudo gpg --dearmor -o /etc/apt/keyrings/envoy-keyring.gpg
```

Comment:
`wget` is more commonly installed in debian-based distros

Reply:
`wget` doesn't pass the content to `stdout` as needed by `gpg`. The file gets stored locally and doesn't get processed by `gpg`. When running `apt update`, it wouldn't find the public key and fail.


Following can work as well:
```bash
wget -O /tmp/signing.key https://apt.envoyproxy.io/signing.key
sudo gpg --dearmor -o /etc/apt/keyrings/envoy-keyring.gpg /tmp/signing.key
```

Reply:
argh - i thought i tested this 8/

Reply:
i think lets do

```console
wget -O- https://apt.envoyproxy.io/signing.key | sudo gpg --dearmor -o /etc/apt/keyrings/envoy-keyring.gpg

```